src/test/resources/søknadDto.json

```diff
@@ -116,10 +116,6 @@
         }
       },
       {
-        "fnr": {
-          "label": "Fødselsnummer eller d-nummer",
-          "verdi": "22042012345678913"
-        },
```

Comment:
Denne tror jeg du må beholde? Den kommer vel fra `LeggTilBarn`, @Assios @miichu  ?

Reply:
Det tallet er et særdeles ugyldig fødselsnummer. Hvis du legger til barn, så har du høyst sannsynlig ikke fødselsnummer, så det burde være mulig å la være å oppgi det.

Reply:
Tenkte mer at "fnr" er/bør være input fra frontend, mens personnummer nedenfor er konseptuelt feil da personnummer er 5 siffer.

Reply:
Opprinnelige skissene hadde datovelger for fødselsdato og personnumer. Fnr-feltet tror jeg ble lagt til fordi jeg påpekte at datovelger + perssonnummer er et dårlig utgangspunkt for å legge inn D-nummer.

Reply:
Oppsummering: 
1. Vi spør aldri om personnnummer 5 siffer i UI. 
2. Vi kan spørre om fødselsnummer/d-nummer, dette valideres i frontend. (se slack samtale)
3. Noen steder har ikke søker fødselsnummer (f.eks annen forelder/tidligere samboer) da kan man fylle inn fødselsdato. Man sender ikke inn begge deler.